db/fixtures/miq_product_features.yml

```diff
@@ -5126,6 +5126,44 @@
       :feature_type: control
       :identifier: configured_system_console
 
+- :name: Automated Workflows
```

Comment:
@agrare We need to come up with an actual name and identifier before we get too far.  I was originally thinking "Embedded Workflows", but I'm tired of the Embedded name for something that's "ours".  Embedded Ansible made sense, but I really never liked "Embedded Automate".

"Automated Workflows" feels redundant, especially when the menu is Automate -> Automated Workflows.

I'm leaning towards just "Workflows".

Thoughts?  @jeffibm what do you think?

Reply:
Not sure if @jeffibm has access rights to your branch, cause he's not a core maintainer - can you give him access?

Reply:
Added

Reply:
> @agrare We need to come up with an actual name and identifier before we get too far. I was originally thinking "Embedded Workflows", but I'm tired of the Embedded name for something that's "ours". Embedded Ansible made sense, but I really never liked "Embedded Automate".
> 
> "Automated Workflows" feels redundant, especially when the menu is Automate -> Automated Workflows.
> 
> I'm leaning towards just "Workflows".
> 
> Thoughts? @jeffibm what do you think?

I agree. I had to use a key named `automated_workflows_workflow` which must be changed.